ChangeLog

```diff
@@ -28,6 +28,7 @@ Bugfix
      In the context of SSL, this resulted in handshake failure. Reported by
      daniel in the Mbed TLS forum. #1351
    * Fix Windows x64 builds with the included mbedTLS.sln file. #1347
+   * Fix Clang Static Analyzer warning reported by aaronmdjones #1002
```

Comment:
The changelog entry should make it clear whether there is a behavior change (true positive from the static analyzer) or not (false positive). There's no behavior change, so I think this should be under “Changes”, not “Bugfix”. Also, please add a full stop at the end of the sentence, and state in which function (or at least in which module) the issue is. I propose:

> Improve the robustness of mbedtls_rsa_rsassa_pkcs1_v15_verify(). This silences a false positive from Clang Static Analyzer reported by aaronmdjones. #1002

Reply:
Thanks @gilles-peskine-arm. I'll update the branch with a fix shortly.